src/gui/Layout.cpp

```diff
@@ -412,59 +375,59 @@ void Layout::ensureRectIsVisible(int x, int y, int width, int height)
 
 XojPageView* Layout::getViewAt(int x, int y)
 {
-	
+
 	XOJ_CHECK_TYPE(Layout);
 
 //  No need to check page cache as the Linear search below starts at cached position.
 //  Keep Binary search handy to check against.
 //	
 //  //Binary Search ... too much overhead makes this a slower option in our use case.
-// 	auto rit = std::lower_bound( this->sizeRow.begin(),  this->sizeRow.end(), y);
-// 	int rb = rit - this->sizeRow.begin();	//get index
-// 	auto cit = std::lower_bound( this->sizeCol.begin(),  this->sizeCol.end(), x);
-// 	int cb = cit - this->sizeCol.begin();
+// 	auto rit = std::lower_bound( this->heightRows.begin(),  this->heightRows.end(), y);
+// 	int rb = rit - this->heightRows.begin();	//get index
+// 	auto cit = std::lower_bound( this->widthCols.begin(),  this->widthCols.end(), x);
+// 	int cb = cit - this->widthCols.begin();
+
 
 
+	auto numRows = this->heightRows.size();//this->mapper.getRows();
+	auto numCols = this->widthCols.size();//this->mapper.getColumns();
```

Comment:
Remove the comments if no longer required.

Reply:
```suggestion
	auto numRows = this->heightRows.size();
	auto numCols = this->widthCols.size();
```